tools/RAiDER/models/model_levels.py

```diff
@@ -505,3 +505,17 @@
     -300,
     -500,
 ]
+
```

Comment:
We should be documenting also A and B for HRRR to stay consistent. Look here: https://rapidrefresh.noaa.gov/faq/HRRR.faq.html

Were the average model height computed using these values?

Reply:
Check the documentation link above, I understand they are separated by the "A.B" for HRRR, equation is provided too?